src/qt/forms/modaloverlay.ui

```diff
@@ -166,6 +166,112 @@
            </item>
           </layout>
          </item>
+         <item>
+          <layout class="QGridLayout" name="gridLayout_3">
+           <property name="horizontalSpacing">
+            <number>15</number>
+           </property>
+           <property name="verticalSpacing">
+            <number>10</number>
+           </property>
+           <item row="1" column="2">
+            <widget class="QToolButton" name="quicksyncoptionsButton">
+             <property name="minimumSize">
+              <size>
+               <width>20</width>
+               <height>20</height>
+              </size>
+             </property>
+             <property name="toolTip">
+              <string>QuickSync ptions</string>
```

Comment:
Possible typo. I believe you wanted line 186 <string>QuickSync ptions</string>  to be                          
                                                                       <string>QuickSync Options</string>.